index.bs

```diff
@@ -221,6 +221,25 @@ and doing cleanup as necessary (see the [[#listen-fully-active]] pattern above).
   but every feature specification should think about this and decide what works best for the feature.
 </div>
 
+<h4 id="non-synchronous-steps">Be aware that navigations can happen between steps running [=in parallel=]</h4>
+While running steps [=in parallel=],
+the document the steps are running in or are otherwise using might be navigated away and (later) navigated to again.
+The steps themselves might not run while the document is not [=fully active=]
+(if they are in a queued task associate with the document),
+and a considerable amount of time might have passed between the steps,
+so the user might not expect some states that are saved in the steps to be preserved.
+
+When writing steps that run [=in parallel=],
+consider also listening to changes to the [=Document/fully active=] state
+and doing cleanup as necessary or even refrain from running some steps
+(see the [[#listen-fully-active]] pattern above).
+<div class=example>
+  APIs that request permission to the user, or requests user interaction through prompt, etc.,
+  should consider re-requesting the user if it detects that the document had become non-fully active.
+</div>
+Note: In theory, steps that run [=in parallel=] should generally already be resilient to changes that might occur between its steps.
```

Comment:
"should" in a note is generally not allowed, but maybe conformance requirements don't apply here? https://infra.spec.whatwg.org/#conformance

Reply:
Ah OK, maybe "are probably already resilient" will be better?

Reply:
"ought to be"

Reply:
Used that now, thanks!